src/pages/Facility/overview.tsx

```
@@ -148,6 +170,63 @@ export function FacilityOverview({ facilityId }: FacilityOverviewProps) {
         </div>
+
+        {/* Pinned Links Section */}
+        {pinnedLinks.length > 0 && (
+          <div>
+            <div className="mb-6 flex items-center justify-between">
+              <h2 className="text-xl font-semibold text-gray-900">
+                {t("pinned_links")}
+              </h2>
+              <DropdownMenu>
+                <DropdownMenuTrigger asChild>
+                  <button className="p-2 rounded-lg hover:bg-gray-100 text-gray-500 hover:text-gray-700 transition-colors">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)